frontend/src/components/Tutorial/TutorialContainer.vue

```diff
@@ -1,396 +1,287 @@
-          address: result?.data.contract_id,
-          contractId: contract.id,
-          defaultState: `{ "have_coin": "True" }`
-        })
-        notify({
-          title: 'OK',
-          text: 'Contract deployed',
-          type: 'success'
-        })
-      } else {
-        notify({
-          title: 'Error',
-          text:
-            typeof result.message === 'string' ? result.message : 'Error Deploying the contract',
-          type: 'error'
-        })
-      }
     }
   },
   {
-    target: '#tutorial-creating-transactions',
+    target: () => '#tutorial-creating-transactions',
     header: {
       title: 'Creating Transactions'
     },
-    content: "This is where you can interact with the deployed contract. You can select a method you want to use from the dropdown, and provide the parameters.  Click “Next” to automatically create a transaction and interact with your deployed contract.",
-    onNextStep: async (store: any, router: any) => {
-      const deployedContract = store.deployedContracts.find((contract: any) => contract.contractId === store.currentContractId)
-      if (deployedContract) {
-        try {
-          const { result } = await rpcClient.call({
-            method: 'call_contract_function',
-            params: [
-              store.currentUserAddress,
-              deployedContract?.address,
-              'WizardOfCoin.ask_for_coin',
-              ['Please give me some coin!']
-            ]
-          })
-
-          if (result?.status === 'success') {
-            if (!store.contractTransactions[deployedContract.address]) {
-              store.contractTransactions[deployedContract.address] = [result]
-            } else {
-              store.contractTransactions[deployedContract.address].push(result)
-            }
-            notify({
-              title: 'OK',
-              text: 'Contract method called',
-              type: 'success'
-            })
-          } else {
-            notify({
-              title: 'Error',
-              text:
-                typeof result.message === 'string' ? result.message : 'Error calling contract method',
-              type: 'error'
-            })
-          }
-
-        } catch (error) {
-          console.error(error)
-          notify({
-            title: 'Error',
-            text: 'Error calling contract method',
-            type: 'error'
-          })
-        }
-      }
+    content: () => "This is where you can interact with the deployed contract. You can select a method you want to use from the dropdown, and provide the parameters.  Click “Next” to automatically create a transaction and interact with your deployed contract.",
+    onNextStep: async () => {
+      tutorialStore.callContractMethod()
+      notify({
+        title: 'OK',
+        text: 'Contract method called',
+        type: 'success'
+      })
     }
   },
   {
-    target: '#tutorial-node-output',
+    target: () => '#tutorial-node-output',
```

Comment:
@bradleySuira why are these changed to functions?

Reply:
fixed removed arrow funds